plugins/rikki/heroeslounge/plugin.yaml

```diff
@@ -122,6 +126,9 @@ permissions:
     rikki.heroeslounge.casterscheduling:
         tab: 'rikki.heroeslounge::lang.plugin.name'
         label: 'Appove casters, assign channels'
+    rikki.heroeslounge.single_playoff:
+        tab: 'rikki.heroeslounge::lang.plugin.name'
+        label: 'Manage Single playoffs and its matches'
```

Comment:
nit: 'single playoff'
Also, I would probably use the word 'individual' over 'single', since one admin could be assigned to multiple playoffs.

Reply:
So I would go with individual_playoffs? And Manage individual playoffs @Fabian-Sommer?